pmd-java/src/main/resources/category/java/errorprone.xml

```diff
@@ -1116,6 +1116,48 @@ public void foo() {
         </example>
     </rule>
 
+    <rule name="DetachedTestCase"
+          language="java"
+          since="6.13"
+          message="Probable detached JUnit test case."
+          class="net.sourceforge.pmd.lang.rule.XPathRule"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_errorprone.html#detachedtestcase">
+        <description>
+The method appears to be a test case since it has public or default visibility, non-static access, no arguments, no return value, has no annotations, but is a member of a class that has one or more JUnit test cases. If it is a utility method, it should likely have private visibility. If it is an ignored test, it should be annotated with @Test and @Ignore.
```

Comment:
Could you please break this into multiple lines? - it would be much easier to read... especially on github, where horizontally scrolling is very difficiult. Thanks!

Reply:
Sure thing!